desc/objectives/_coils.py

```diff
@@ -835,6 +835,15 @@ def body(k):
         return min_dist_per_coil
 
 
+def _copy_rpz_periods(rpz, NFP):
```

Comment:
this is done in another PR as well right? the distance one?

Reply:
Yeah whichever gets second will need to be rebased a bit